diff --git a/intellihide.js b/intellihide.js
index d9a9d99..6c919cd 100644
--- a/intellihide.js
+++ b/intellihide.js
@@ -115,7 +115,7 @@ var Intellihide = class HideTopBar_Intellihide {
             this._checkOverlap.bind(this)
         ], [
             // update wne monitor changes, for instance in multimonitor when monitor are attached
-            Meta.MonitorManager.get(),
+            global.backend.get_monitor_manager?.() ?? Meta.MonitorManager.get(),
             'monitors-changed',
             this._checkOverlap.bind(this)
         ]);
diff --git a/metadata.json b/metadata.json
index dfaaaca..44b52dd 100644
--- a/metadata.json
+++ b/metadata.json
@@ -8,9 +8,10 @@
     "40",
     "41",
     "42",
-    "43"
+    "43",
+    "44"
   ],
   "url": "https://gitlab.gnome.org/tuxor1337/hidetopbar",
   "uuid": "hidetopbar@mathieu.bidon.ca",
   "version": 112
-}
\ No newline at end of file
+}

