diff --git a/src/libutil/util.cc b/src/libutil/util.cc
index 28df30fef..7c1b8c597 100644
--- a/src/libutil/util.cc
+++ b/src/libutil/util.cc
@@ -585,7 +585,7 @@ Path getHome()
                     homeDir.reset();
                 }
             } else if (st.st_uid != geteuid()) {
-                warn("$HOME ('%s') is not owned by you, falling back to the one defined in the 'passwd' file", *homeDir);
+                warn("$HOME ('%s') is not owned by you, falling back to the one defined in the 'passwd' file. home owend by uid %d, you are uid %d", *homeDir, st.st_uid, geteuid());
                 homeDir.reset();
             }
         }

