diff --git a/src/printer.rs b/src/printer.rs
index 6b8cbc3..a20e3bb 100644
--- a/src/printer.rs
+++ b/src/printer.rs
@@ -318,12 +318,7 @@ impl<'a> Printer for InteractivePrinter<'a> {

         writeln!(
             handle,
-            "{}{}{}",
-            description
-                .kind()
-                .map(|kind| format!("{}: ", kind))
-                .unwrap_or_else(|| "".into()),
-            self.colors.filename.paint(description.title()),
+            "{}",
             mode
         )?;
