diff --git a/package/helium.desktop b/package/helium.desktop
index 8cce3b1..cd43a0c 100644
--- a/package/helium.desktop
+++ b/package/helium.desktop
@@ -3,7 +3,7 @@ Version=1.0
 Name=Helium
 GenericName=Web Browser
 Comment=Access the Internet
-Exec=helium %U
+Exec=helium-bin %U
 StartupNotify=true
 StartupWMClass=helium
 Terminal=false
@@ -15,8 +15,8 @@ Actions=new-window;new-private-window;
 
 [Desktop Action new-window]
 Name=New Window
-Exec=helium
+Exec=helium-bin
 
 [Desktop Action new-private-window]
 Name=New Incognito Window
-Exec=helium --incognito
+Exec=helium-bin --incognito

